activestorage/test/test_helper.rb

```diff
@@ -14,15 +13,100 @@
 
 require "active_record/testing/query_assertions"
 
-require "active_job"
+require "rails"
+require "active_record/railtie"
+require "active_storage/engine"
+
+ActiveStorage::FixtureSet.file_fixture_path = File.expand_path("fixtures/files", __dir__)
+ActiveStorage.verifier = ActiveSupport::MessageVerifier.new("Testing")
+
+module ActiveStorage
+  class TestApp < Rails::Application
```

Comment:
Why we need this app? I don't see it used anywhere.

Reply:
The tests are heavily dependent upon `ActiveStorage::Blob` so we need to initialize AR, and using the [engine](https://github.com/rails/rails/blob/main/activestorage/lib/active_storage/engine.rb) does all of that for us. We also load test models, migrations, fixtures, etc are local to this application (which "lives" under `test/support`).

Other than generating an application on the fly, I couldn't think of an alternative. Do you have any suggestions?

Reply:
Another example where the initialized application is used:
https://github.com/rails/rails/blob/0add5dba834f2f1b84fcf1bd1b758545b325fb73/activestorage/test/urls/rails_storage_redirect_url_test.rb#L7

My goal with this PR is to remove the dummy app, I see one of two options:

* Use `Rails::Generators::AppGenerator` to build an app that we throw away
* Create a TestApp class and initialize it in the test helper

This PR takes the second approach, but I'm open to suggestions.